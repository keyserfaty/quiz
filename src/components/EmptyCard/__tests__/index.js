import React from 'react'
import { View, Text } from 'react-native'

import Card from '../index'

import Shallow from 'react-test-renderer/shallow'

const renderer = new Shallow()

test('Should render', () => {
  let component = renderer.render(<Card />)

  expect(component).toMatchSnapshot('initial render')
})

test('Should display a different style', () => {
  let component = renderer.render(<Card style={{ prop: true }} />)

  expect(component).toMatchSnapshot('initial render')
})

test('Should display some children', () => {
  let component = renderer.render(
    <Card>
      <View>
        <Text>Some children</Text>
      </View>
    </Card>
  )

  expect(component).toMatchSnapshot('initial render')
})
