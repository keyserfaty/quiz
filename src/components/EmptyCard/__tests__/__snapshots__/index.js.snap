// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should display a different style: initial render 1`] = `
<Component
  style={
    Array [
      Object {
        "backgroundColor": "white",
        "borderRadius": 3,
        "elevation": 5,
        "marginBottom": 8,
        "paddingHorizontal": 16,
        "paddingVertical": 24,
        "shadowColor": "rgba(0,0,0,0.5)",
        "shadowOffset": Object {
          "height": 2,
          "width": 0,
        },
        "shadowOpacity": 11,
        "shadowRadius": 2,
        "width": "100%",
      },
      Object {
        "prop": true,
      },
    ]
  }
/>
`;

exports[`Should display some children: initial render 1`] = `
<Component
  style={
    Array [
      Object {
        "backgroundColor": "white",
        "borderRadius": 3,
        "elevation": 5,
        "marginBottom": 8,
        "paddingHorizontal": 16,
        "paddingVertical": 24,
        "shadowColor": "rgba(0,0,0,0.5)",
        "shadowOffset": Object {
          "height": 2,
          "width": 0,
        },
        "shadowOpacity": 11,
        "shadowRadius": 2,
        "width": "100%",
      },
      undefined,
    ]
  }
>
  <Component>
    <Component>
      Some children
    </Component>
  </Component>
</Component>
`;

exports[`Should render: initial render 1`] = `
<Component
  style={
    Array [
      Object {
        "backgroundColor": "white",
        "borderRadius": 3,
        "elevation": 5,
        "marginBottom": 8,
        "paddingHorizontal": 16,
        "paddingVertical": 24,
        "shadowColor": "rgba(0,0,0,0.5)",
        "shadowOffset": Object {
          "height": 2,
          "width": 0,
        },
        "shadowOpacity": 11,
        "shadowRadius": 2,
        "width": "100%",
      },
      undefined,
    ]
  }
/>
`;
